packages/domain-picker/src/domain-picker/domain-i-own-item.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { __ } from '@wordpress/i18n';
+import { Icon, chevronRight } from '@wordpress/icons';
+import { createInterpolateElement } from '@wordpress/element';
+import { useLocalizeUrl } from '@automattic/i18n-utils';
+
+/**
+ * Internal dependencies
+ */
+
+interface Props {
+	siteSlug: string;
+	source: string;
+}
+
+const DomainIOwn: React.FunctionComponent< Props > = ( { siteSlug, source } ) => {
+	const localizeUrl = useLocalizeUrl();
+
+	return (
+		<div className="domain-picker__suggestion-item contains-link">
+			<div className="domain-picker__suggestion-item-name">
+				<span className="domain-picker__domain-wrapper with-margin">
+					{ createInterpolateElement(
+						__( '<strong>Already own a domain?</strong>', __i18n_text_domain__ ),
+						{
+							strong: <strong />,
+						}
+					) }
+				</span>
+				<div>
+					<span className="domain-picker__item-tip">
+						{ __( "You can use it as your site's address", __i18n_text_domain__ ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'You can use it as your site's address.', { context: 'Upgrades: Register domain description'} )` *ES Score: 8*